tests/manage/mcg/conftest.py

```diff
@@ -26,3 +26,14 @@ def pytest_collection_modifyitems(items):
                     f" mcg is not supported on {config.ENV_DATA['platform'].lower()}"
                 )
                 items.remove(item)
+
+    if config.DEPLOYMENT.get("disconnected"):
+        for item in items.copy():
+            if any(
+                cloud_platform in item.name.upper()
+                for cloud_platform in ["AWS", "AZURE", "GCP", "IBM"]
```

Comment:
Wouldn't be possible to use [`constants.CLOUD_PLATFORMS`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L723) here?

Reply:
Addressed :)